README.md

```diff
@@ -9,21 +9,21 @@ Symfony and API Platform.
 
 Install composer dependencies:
 
-```
+```sh
```

Comment:
```suggestion
```console
```
Console will be even better: https://stackoverflow.com/a/49817279

Reply:
I changed it to `console`
But now PhpStorm does not recognize it.
I guess i can open an issue on PhpStorm.